README.md

```diff
@@ -82,6 +83,93 @@ This project is in the worldwide [public domain](LICENSE.md). As stated in [CONT
 >
 > All contributions to this project will be released under the CC0 dedication. By submitting a pull request, you are agreeing to comply with this waiver of copyright interest.
 
-### Origin 
+### Origin
 
 This project was originally forked from [18F](https://github.com/18f/pulse) and has been modified to fit the Canadian context.
+
+## ---------------------------------------------------------------------
+
+## Faire le suivi du respect des pratiques en matière de sécurité Web par les domaines du gouvernement du Canada
+
+Ce dépôt est l’un des éléments du _Suivre la conformité en matière de sécurité Web_, une application Web qui analyse les sites Web du gouvernement du Canada et indique en quoi ils respectent les bonnes pratiques en matière de sécurité Web, comme l’énonce l'[Avis de mise en œuvre de la Politique sur la technologie de l’information (AMPTI) : Mise en œuvre de HTTPS pour les connexions Web sécurisées](https://www.canada.ca/fr/secretariat-conseil-tresor/services/technologie-information/avis-mise-oeuvre-politique.html). `track-web` est une application Web qui affiche les résultats du  [tracker](https://github.com/cds-snc/tracker), l’analyseur de domaines.
+
+Voici à quoi cela ressemble avec les données de démonstration :
+
+|         |
+|---------|
+|![French landing page: header with title, some text, and a chart showing number of domains that enforce HTTPS](/docs/img/fr-landing.png) |
```

Comment:
Bit of a nit pick, but should the description of the french images be in English?

Reply:
yeah, they should, good catch. I'll merge this now and change this later (translations are endless 😅 )